data_models/contents.py

```diff
@@ -7,24 +7,24 @@
 class Contents:
     """This class creates content_list which is list of extracted contents"""
 
+    CONTENTS_DICT_LIST = []
+
     def __init__(self):
         self.content_list = list()
 
     def add_content(self, _content):
         """This method appends extracted content to content_list"""
-        self.content_list.append(_content)
-
-    def get_formatted_list(self):
-        """This method converts the python class object list
-         into dictionary list"""
 
-        return_list = []
+        self.content_list.append(_content)
+        self.__dict__['content_list'].remove(_content)
+        self.__dict__.update({'content_list': self.updated_dict(_content)})
```

Comment:
I guess you can directly `update` without `remove`

Reply:
`remove` is to remove the class object entry which we don't need. So we keep `remove` for now. We can consider refactoring in future iterations.